source/data/Symbol_info.rst

```diff
@@ -89,3 +92,83 @@ Crypto
 	BTCUSD → OKEX:BTCUSD // crypto pair
 	BTCUSDTPERP → OKEX:BTCUSDTPERP // perpetual swap contract
 	BTCUSDT25H2022 → BITMEX:BTCUSDT25H2022 // futures contract
+
+
+Price display
+......................
+
+To manage how the price displays on the chart, use the following parameters in `/symbol_info`_.
```

Comment:
~To manage how the price displays on the chart~
To manage how the price is displayed on the chart